### ISSUE TITLE
[Question] Guidance about the purescript tool set chain

### ISSUE THREAD

**Issue by PierreR** [purescript/psc-package#64]
My understanding so far is that `psc-package` is meant to replace `bower` . On the other hand `pulp` is still useful. Is that correct ? Is it possible to use `psc-package` together with `purs` without requiring a third tool ?

`PureScript by Example` does not mention `psc-package` at all and would suggest the use of  `bower` to tackle dependencies. Does it mean that `psc-package` is not ready for broad user consumption yet ? 

At this point, I have found the `psc-package` story quite confusing. What seems to be missing is a section about simple usage patterns. I have managed to build a project with `psc-package` but it is still unclear to me if I should `bundle` using `purs` or `pulp`. 

Thanks for your help.

**Comment by PierreR** [comment]
I have found out that one thing that works well is the combo `pulp` & `psc-package`. I don't know if this is the idiomatic way but if it is would you be open to a PR that add the mention of `pulp` in the README  ?

**Comment by kritzcreek** [comment]
I think pulp already supports psc-package according to https://github.com/purescript-contrib/pulp#dependency-management.

**Comment by PierreR** [comment]
@kRITZCREEK Yep it does and it works pretty well for me so far. I am just saying that if I knew I had to use `psc-package` together with `pulp` it would have save me hours of troubleshooting. Somehow the fact that one of the goal of the project is to remove the need of node has been a source of confusion as well (as I then assume `pulp` + `psc-package` was not the way to go). 

Anyhow mentioning `pulp` on the README or some other directions about how `psc-package` is meant to be used with other tools  seems like a good idea.

**Comment by paf31** [comment]
You don't need to use Pulp, but a documentation fix to mention it briefly would be a good idea, yes.

**Comment by PierreR** [comment]
@paf31 What would be the replacement for `pulp build -O --to output.js`  in a `psc-package` setting without `pulp` ?

**Comment by paf31** [comment]
You would use `psc-package build` followed by `purs-bundle` on the output.

**Comment by arrowd** [comment]
How do I use `purs-bundle` on the output? I tried `psc-package sources | xargs purs bundle`, but that didn't work. Why there is no `psc-package bundle` in the first place?

**Comment by JordanMartinez** [comment]
@arrowd You should probably be using [`spago`](https://github.com/spacchetti/spago/) instead of `psc-package`. `spago` more or less succeeds `psc-package` (see spacchetti/spago#423). Then, you could use `spago bundle-app` or `spago bundle-module` depending on what you want to do.

As for your question, I don't know because I don't use that tool anymore.

**Comment by arrowd** [comment]
@JordanMartinez Thanks for the pointer, I'll try this out.

It would be nice, BTW, if `psc-package` or `spago` would be mentioned in https://github.com/purescript/documentation/blob/master/guides/PureScript-Without-Node.md
